newsroom/wire/views.py

```diff
@@ -134,6 +134,38 @@ def delete_dashboard_caches():
         app.cache.delete(f"{HOME_ITEMS_CACHE_KEY}{company['_id']}")
 
 
+def get_personal_dashboards_data(user, company, topics):
+    section_filters = get_resource_service("section_filters").get_section_filters_dict()
+
+    def get_topic_items(topic):
+        query = superdesk.get_resource_service("wire_search").get_topic_query(topic, user, company, section_filters)
```

Comment:
Should we also provide a `size` param as to limit the number of items returned for these homepage cards?

Reply:
I have it set in the `get_items_by_query` method